.github/workflows/python-tests.yml

```diff
@@ -1,6 +1,5 @@
 on:
   push:
-    branches: [ main ]
   pull_request:
```

Comment:
After doing that, I've doubt, are there any cases in which the `pull_request` event triggers but not the `push` one? I mean, maybe with the "on push" event we are able to cover both cases.

Reply:
> ne? I mean, maybe with the "on push" event we are able to cover both case

Yes, have sense

we reduce to "on push" trigger

Reply:
I will invalidate this pull request and you put a new one

Reply:
Okay @andrea-mucci, I'll close this one and create an new one 🙂